squad/frontend/templates/squad/_builds_table.html

```diff
@@ -1,62 +1,61 @@
 {% load humanize %}
 {% load squad %}
+<h3> Builds </h3>
```

Comment:
This line adds redundant text to the page

Reply:
I think the actual builds table should be its own file and it then gets included and a title added. I realize this is a little bit too much work so instead, perhaps, it is best to remove "Latest Builds" and lave builds there.